lib/generators/simple_form/templates/config/initializers/simple_form_bootstrap.rb

```diff
@@ -1,166 +1,439 @@
 # frozen_string_literal: true
-#
+
+# Please do not make direct changes to this file!
+# This generator is maintained by the community around simple_form-bootstrap:
+# https://github.com/rafaelfranca/simple_form-bootstrap
+# All future development, tests, and organization should happen there.
+# Background history: https://github.com/plataformatec/simple_form/issues/1561
+
 # Uncomment this and change the path if necessary to include your own
 # components.
-# See https://github.com/plataformatec/simple_form#custom-components to know
-# more about custom components.
+# See https://github.com/plataformatec/simple_form#custom-components
+# to know more about custom components.
 # Dir[Rails.root.join('lib/components/**/*.rb')].each { |f| require f }
-#
+
 # Use this setup block to configure all options available in SimpleForm.
 SimpleForm.setup do |config|
+  # Default class for buttons
+  config.button_class = 'btn'
```

Comment:
Should this be `'btn btn-primary'` or `'btn btn-default'`?

Reply:
Answered here: https://github.com/rafaelfranca/simple_form-bootstrap/pull/56#discussion_r183664638